content/sensu-go/5.14/getting-started/learn-sensu.md

```diff
@@ -36,38 +36,37 @@ ENABLE_SENSU_SANDBOX_PORT_FORWARDING=1 vagrant up
 {{< /highlight >}}
 
 The Learn Sensu sandbox is a CentOS 7 virtual machine pre-installed with Sensu, InfluxDB, and Grafana.
-It is intended for use as a learning tool; we do not recommend this tool as part of a production installation.
-To install Sensu in production, please see the [installation guide](../../installation/install-sensu).
-The sandbox startup process takes about five minutes.
+It's intended for you to use as a learning tool--we do not recommend using it in a production installation. To install Sensu in production, use the [installation guide](../../installation/install-sensu) instead.
 
-_NOTE: The sandbox configures VirtualBox to forward TCP ports 3002 and 4002 from the sandbox virtual machine to the localhost to make it easier for you to interact with the sandbox dashboards. Dashboard links provided in this tutorial assume port forwarding from the VM to the host is active._
+The sandbox startup process takes about 5 minutes.
+
+_NOTE: The sandbox configures VirtualBox to forward TCP ports 3002 and 4002 from the sandbox virtual machine (VM) to the localhost to make it easier for you to interact with the sandbox dashboards. Dashboard links provided in this tutorial assume port forwarding from the VM to the host is active._
 
 **4. SSH into the sandbox**
 
-Thanks for waiting! To start using the sandbox:
+Thanks for waiting! To start shell access to the sandbox:
```

Comment:
This reads sorta odd to me. Maybe `To start shell into the sandbox`?  Or `To start your shell access to the sandbox`.